ledger/participant-integration-api/src/main/resources/db/migration/h2database-appendonly/V1__Append_only_schema.sql

```diff
@@ -42,7 +42,7 @@ CREATE TABLE packages (
     package_id VARCHAR PRIMARY KEY NOT NULL,
     upload_id VARCHAR NOT NULL,
     source_description VARCHAR,
-    size BIGINT NOT NULL,
+    siz BIGINT NOT NULL,
```

Comment:
nitpic: siz looks bit odd, can we go with something like package_size instead?

Reply:
or payload_size

Reply:
renamed